src/Badge.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
```

Comment:
Trivial, believe `Standard` linting picks up a warning for the extra new-line space in lines 6-7. Please consider updating code format?
<img width="566" alt="screen shot 2017-10-02 at 8 42 59 pm" src="https://user-images.githubusercontent.com/6441326/31105460-5c220426-a7b2-11e7-9b8e-4dc3ae57ba71.png">